.travis.yml

```diff
@@ -94,7 +94,9 @@ install:
 
 before_script:
   - travis_retry ./testing/downloadSpark.sh $SPARK_VER $HADOOP_VER
-  - ./testing/setupLivy.sh
+  - if [[ -n $LIVY_VER ]]; then ./testing/downloadLivy.sh $LIVY_VER; fi
+  - if [[ -n $LIVY_VER ]]; then export LIVY_HOME=`pwd`/livy-server-$LIVY_VER; fi
+  - if [[ -n $LIVY_VER ]]; then export SPARK_HOME=`pwd`/spark-$SPARK_VER-bin-hadoop$HADOOP_VER; fi
```

Comment:
is this going to cause problem exporting SPARK_HOME here and then the next line L100 in zeppelin-env.sh?
doesn't zeppelin-env.sh always win?

I think the better model is "if Livy" add "export" to conf/zeppelin-env.sh instead of relying on env set in travis.yml